src/scss/components/_form.scss

```diff
@@ -7,6 +7,34 @@
 **/
 
 .form {
+  position: relative;
```

Comment:
@MarcoBrunoBR We had an absolute positioned element before this version, you are right, we don't need it anymore 👍